app/graphql/resolvers/top_volunteer_resolver.rb

```diff
@@ -0,0 +1,79 @@
+module TopVolunteerResolver
```

Comment:
Typically a resolver would be responsible for a single resource/query type, and the use of query params would control the filtering of data.

In this case I would suggest calling the Module `VolunteerResolver` and add orderby and pagination. This will support getting the top N volunteers and much more.

Reply:
Good point, we added some comments to add some context of this existence and also will change the name!

Reply:
Just want to provide a bit more clarity on this topic. This resolver was created to return a list of users ordered by volunteer hours. Unlike the `UserResolver`, this resolver doesn't just return a list of users. Instead, it tries to expose a concept of "user-hours" based on users and events. This concept isn't well represented in code but is used in a couple of places like the top volunteers dashboard and the admin reporting function.